docs/adding_packages/packaging_policy.md

```diff
@@ -113,9 +113,36 @@ for each combination. There are some particular cases for this general rule:
 Recipes can list any number of options with any meaning, and defaults are up to the recipe itself. The CI cannot enforce anything
 in this direction. However, there are a couple of options that have a special meaning for the CI.
 
+### Recommended Names
+
+It's often needed to add options to toggle specific library features on/off. Regardless of the default, there is a strong preference for using positive naming for options. In order to avoid the fragmentation, we recommend to use the following naming conventions for such options:
```

Comment:
```suggestion
Adding options is often needed to toggle specific library features on/off. Regardless of the default, there is a strong preference for using positive naming for options. In order to avoid the fragmentation, we recommend using the following naming conventions for such options:
```